src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -139,7 +139,7 @@ class Test
 {
     void Method()
     {
-        var t1 = [||](a: 1, b: 2);
+        var t1 = [||](a: 1, B: 2);
```

Comment:
plz don't update test inputs.  Either update the outputs, or add new tests to cover these cases if you care about them.